src/napari_sphinx_theme/assets/napari.tsx

```diff
@@ -505,6 +505,20 @@ function addAutoLoopAndControlsToVideos() {
   });
 }
 
+/**
+ * Fixes issue with admonition dropdowns showing empty white space when hidden:
+ * https://github.com/napari/napari-sphinx-theme/issues/99
+ */
+function renderDropdownButtonIcons() {
+  const buttons = Array.from(
+    document.querySelectorAll('.dropdown.admonition .toggle-button'),
+  );
+
+  for (const button of buttons) {
+    render(<PlusIcon />, button);
+  }
```

Comment:
i dont exactly know enough about react to understand what's going on here, is it inserting the SVG into the button div or something? could you explain how this fixes the issue?